### DISCUSSION TITLE
Trouble with ETH pool

### DISCUSSION THREAD

**Discussion by UberDudePL** [coinfoundry/miningcore#961] [Q&A] [answered]
Hello,

First of all, big thanks to the whole team making this happen!

I have a trouble with finishing my pool setup. I managed to run geth and miningcore but I cannot connect to it with Gminer. I managed to connect with Pheonixminer but it didn't mine at all. 

Here is my config.json (had to change to txt to upload)
[config.txt](https://github.com/coinfoundry/miningcore/files/7224419/config.txt)

Here is log from miningcore were you can see that miner is trying to connect but connection get closed:
[core.log](https://github.com/coinfoundry/miningcore/files/7224440/core.log)

Screen of gminer error
![gminer](https://user-images.githubusercontent.com/91315950/134648142-8f219270-ef8d-4282-82a1-7405602f7c10.png)

command I run geth with:
geth --datadir=/mnt/Pool2 --syncmode=fast --cache=21000 --maxpeers=250 --txpool.globalslots=1000 --http --miner.etherbase='0x10fff36f8e17EC62D8c37692cc9eb2A670CD155a' --mine --miner.threads=0 --miner.extradata='UberPool'

When I connect with Pheonix miner mothing happens no matter how many minutes passes. Miner Hashrate is zero and it does not receive jobs

Can you please support me? Most probably I am missing something

Please let me know if anything else is needed to provide

Many thanks in advance

**Comment by oliverw** [comment, marked as answer]
The most likely issue is this: https://github.com/coinfoundry/miningcore#ethereum

**Reply by UberDudePL** [reply]
Does it mean I have to add something to config.json. Apologies but I am beginner in this area

Thanks

**Reply by oliverw** [reply]
> Does it mean I have to add something to config.json. Apologies but I am beginner in this area

No, it means that you should configure your miner accordingly or chose a different one.

**Reply by UberDudePL** [reply]
OK but what about not sending jobs? when I connected with another miner?

**Reply by seperatis** [reply]
Add this to then end of your Phoenix Miner config.

`-proto 4`

You need to tell the miner to use Ethereum Stratum 1.0 (Nicehash). For Phoenix, you add the above to your config. For other miners you must find out the way.

**Reply by UberDudePL** [reply]
Awesome! It works!! Thank you!

**Reply by UberDudePL** [reply]
for Gminer it was --proto stratum

**Reply by UberDudePL** [reply]
@oliverw and @seperatis would you be so kind and assist me with API as well? When I try to setup it I get the error:
![image](https://user-images.githubusercontent.com/91315950/134659701-73c6dd05-e293-4b5c-ba8d-372a41974c0f.png)

**Reply by UberDudePL** [reply]
Solved this by changing listingIP of API to "*"
